### ISSUE TITLE
Pickaxe not the effective tool anymore on stone blocks and alike

### ISSUE THREAD

**Issue by Norrox** [HyperPigeon/Eldritch-Mobs#79]
When i enable this mod, there seems to modify the blocks somehow?
pickaxe is useless agains all stone blocks.

Note,
i have commented out "EldritchMobsBlocks.init();" and i run this mod on server side only.

any ideas? :)

**Comment by HyperPigeon** [comment]
I wasn't able to replicate this error, are you using the latest version of Eldritch Mobs?

**Comment by Norrox** [comment]
Yeah, i downloaded the latest comitted version here on git and made myself a new jar after commenting out that line. 
It works when i use the regular file downloaded from curse

**Comment by HyperPigeon** [comment]
That's a bit strange. Just to clarify, the version you downloaded from curseforge is working normally but the version that you built from the latest committed version on GitHub doesn't?
